mlir/utils/jenkins/perfRegressionReport.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/evn python3
+
+import reportUtils
+
+from pathlib import PurePath
+import sys
+from typing import Tuple
+
+import pandas as pd
+
+def loadMlirData(filename):
+    df = pd.read_csv(filename, sep=',', header=0, index_col=False)
+    df.drop(columns=['MIOpen TFlops'], inplace=True, errors='ignore')
+    return df
+
+def computePerfStats(oldDf: pd.DataFrame, newDf: pd.DataFrame) -> Tuple[pd.DataFrame, pd.DataFrame]:
+    data = newDf.merge(oldDf, on=reportUtils.TEST_PARAMETERS, suffixes=('_new', '_old'))
+    data.rename(columns={'TFlops_old': 'TFlops (old)', 'TFlops_new': 'TFlops (new)'}, inplace=True)
```

Comment:
Is it possible to show the date when data was collected instead of 'old' and 'new'?

Reply:
It should be possible, though I'm not sure what the best way to go about it is. One idea would be to add a `perf-run-date` file to the artifact and to read from it if possible.